addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -92,55 +93,55 @@ Object.assign(WelcomeScreen, {
     template: 'website.Configurator.WelcomeScreen',
 });
 
+class IndustrySelectionAutoComplete extends AutoComplete {
+    static timeout = 400;
+
+    get dropdownOptions() {
+        return {
+            ...super.dropdownOptions,
+            position: "bottom-fit",
+        }
+    }
+
+    get ulDropdownClass() {
+        return `${super.ulDropdownClass} custom-ui-autocomplete shadow-lg border-0 o_configurator_show_fast o_configurator_industry_dropdown`
+    }
+}
+
 class DescriptionScreen extends Component {
     setup() {
         this.industrySelection = useRef('industrySelection');
         this.state = useStore();
-        this.labelToId = {};
-        this.autocompleteHasResults = true;
         this.orm = useService('orm');
 
         onMounted(() => this.onMounted());
     }
 
     onMounted() {
         this.selectWebsitePurpose();
-        $(this.industrySelection.el).autocomplete({
-            appendTo: '.o_configurator_industry_wrapper',
-            delay: 400,
-            minLength: 1,
-            source: this._autocompleteSearch.bind(this),
-            select: this._selectIndustry.bind(this),
-            classes: {
-                'ui-autocomplete': 'custom-ui-autocomplete shadow-lg border-0 o_configurator_show_fast o_configurator_industry_dropdown',
-            },
-        });
-        if (this.state.selectedIndustry) {
-            this.industrySelection.el.value = this.state.selectedIndustry.label;
-            this.industrySelection.el.parentNode.dataset.value = this.state.selectedIndustry.label;
-            this.labelToId[this.state.selectedIndustry.label] = this.state.selectedIndustry.id;
-        }
     }
-
     /**
      * Set the input's parent label value to automatically adapt input size
      * and update the selected industry.
      *
      * @private
-     * @param {String} label an industry label
+     * @param {Object} suggestion an industry
      */
-    _setSelectedIndustry(label) {
-        this.industrySelection.el.parentNode.dataset.value = label;
-        if (!this.autocompleteHasResults) {
-            // Unknown industry.
-            this.state.selectIndustry(label, -1);
-        } else {
-            const id = this.labelToId[label];
-            this.state.selectIndustry(label, id);
-        }
+    _setSelectedIndustry(suggestion) {
+        const { label, id } = Object.getPrototypeOf(suggestion);
+        this.state.selectIndustry(label, id);
         this.checkDescriptionCompletion();
     }
 
+    get sources() {
+        return [
+            {
+                options: (request) => {
+                    return request.length < 1 ? [] : this._autocompleteSearch(request);
+                },
+            },
+        ];
+    }
```

Comment:
This is to replace the `minLength: 1,` from jQuery, right?
Isn't that part of the owl `AutoComplete` implementation?

Reply:
By default, the owl `AutoComplete` doesn't have this limit. So it's 0 for now.